### PR TITLE
docs(*): update markdown files and fix typos - I257

### PR DESCRIPTION
**Pull request by michizhou**
Signed-off-by: Michael Zhou <myz227@nyu.edu>

# Issue #257 
<OVERALL SUMMARY OF PULL REQUEST>
Updated markdown files by correcting for incorrect markdown links, integrating existing links into new markdown, and ensuring consistency in markdown style. Also fixes typos, grammar, and punctuation errors in documentation. All tests confirmed to pass.

### Changes
- Corrected wrong markdown link for Code of Conduct in `CHARTER.md`
  - Added `/code-of-conduct/` to end of existing markdown link
- Replaced in-text URLs with appropriate markdown to eliminate visibility from documentation
- Capitalized first letter for numbered bullet points to stay consistent with style
- Modified markdown to be more appropriate with content of associated links

### CODE REVIEW

File: CHARTER.md [hunk 1]
```diff
@@ -31,21 +31,21 @@ f. The TSC may elect a TSC Chair, who will preside over meetings of the TSC and
 
 g. Responsibilities: The TSC will be responsible for all aspects of oversight relating to the Project, which may include:
 
-i. coordinating the technical direction of the Project;
+i. Coordinating the technical direction of the Project;
```
Comment (irmerk) [inline CHARTER.md]: All changes to `CHARTER.md` can be removed.

**Comment by michizhou** [comment]
@irmerk Which of Accord Project's repos are considered main repos? Would it just be the pinned repos, or more than that? Given that there are 30 repos within the organization, it would be best if you could give me a list of all the main repos so that I may open similar PRs in each of them.

**Comment by irmerk** [comment]
Ah yes, we need to do a bit of cleanup, but they are basically those listed in the ecosystem for [techdocs](https://github.com/accordproject/techdocs):

- ergo
- cicero
- concerto
- markdown-transform
- markdown editor
- cicero-ui
- concerto-ui
- template-studio-v2
- vscode-extension
- models
- cicero-template-library
- techdocs

**Comment by michizhou** [comment]
@irmerk I removed all the changes to `CHARTER.md` as you suggested. I am in the process of creating PRs for all the repos you listed, so look forward to those. Let me know if further changes are needed for this PR in the meantime.

File: CHARTER.md [hunk 2]
```diff
@@ -59,7 +59,7 @@ d. In the event a vote cannot be resolved by the TSC, any voting member of the T
 ## 4. Compliance with Policies
 a. This Charter is subject to the Series Agreement for the Project and the Operating Agreement of LF Projects. Contributors will comply with the policies of LF Projects as may be adopted and amended by LF Projects, including, without limitation the policies listed at https://lfprojects.org/policies/.
 
-b. The TSC may adopt a code of conduct (“CoC”) for the Project, which is subject to approval by the Series Manager. In the event that a Project-specific CoC has not been approved, the LF Projects Code of Conduct listed at https://lfprojects.org/policies will apply for all Collaborators in the Project.
+b. The TSC may adopt a Code of Conduct (“CoC”) for the Project, which is subject to approval by the Series Manager. In the event that a Project-specific CoC has not been approved, the [LF Projects Code of Conduct](https://lfprojects.org/policies/code-of-conduct/) will apply for all Collaborators in the Project.
```
Comment (irmerk) [inline CHARTER.md]: Still have changes to remove from `CHARTER.md`

File: CHARTER.md [hunk 3]
```diff
@@ -77,16 +77,16 @@ c. Under no circumstances will LF Projects be expected or required to undertake
...
-ii. respect the rights of all trademark owners, including any branding and trademark usage guidelines.
+ii. Respect the rights of all trademark owners, including any branding and trademark usage guidelines.
 
 ## 7. Intellectual Property Policy
 a. Collaborators acknowledge that the copyright in all new contributions will be retained by the copyright holder as independent works of authorship and that no contributor or copyright holder will be required to assign copyrights to the Project.
 
 b. Except as described in Section 7.c., all contributions to the Project are subject to the following:
 
-i. All new inbound code contributions to the Project must be made using the Apache License, Version 2.0, available at https://www.apache.org/licenses/LICENSE-2.0 (the “Project License”).
+i. All new inbound code contributions to the Project must be made using the [Apache License, Version 2.0](https://www.apache.org/licenses/LICENSE-2.0) (the “Project License”).
```
Comment (irmerk) [inline CHARTER.md]: Still have changes to remove from `CHARTER.md`

**Comment by irmerk** [comment]
@michizhou great, I'll approve and merge all of them at the same time so it's just easier that way.

**Comment by irmerk** [comment]
@michizhou just making sure you remember the `CHARTER` changes in this one.

**Comment by michizhou** [comment]
@irmerk I have reverted all changes to `CHARTER` such that the file won't be changed by this PR. Feel free to let me know if you need anything else before merging this. Thanks!

**Pull request merged**